tests/unit/test_install.py

```diff
@@ -1262,6 +1264,7 @@ def test_runs_upgrades_on_too_old_version(ws, any_prompt):
     sql_backend = MockBackend()
     wheels = create_autospec(WheelsV2)
     install.run(
+        default_config=WorkspaceConfig(inventory_database='ucx'),
```

Comment:
Can you rewrite the logic the way that existing tests won't have to be modified to keep working with the new functionality? 

There's a high probability of hidden bugs if changes here are required.